completion.bash

```diff
@@ -42,10 +42,28 @@ _charmcraft()
             COMPREPLY=( $(compgen -W "--help" -- "$cur") )
             return
             ;;
+
+        # store related commands
```

Comment:
misplaced comment is misplaced

Reply:
indeed, fixed, thanks!

Reply:
FTR, I removed the "message as a separation/grouping", as I like to keep more simple (grouping everything that doesn't have anything else than --help)